mrmustard/utils/parametrized.py

```diff
@@ -22,10 +22,9 @@
 
 
 class Parametrized:
-    r"""
-    Abstract base class for all parametrized objects (gates, detectors, etc...)
+    r"""Abstract base class for all parametrized objects (gates, detectors, etc...)
 
-    Arguments (must be all called with keyword):
+    Args (must be all called with keyword):
         For each trainable parameter:
         xxx (tensor): initial value
         xxx_bounds (float, float): numerical bounds. Default is (None, None) for unbounded parameters
```

Comment:
Should be written as `Keyword arguments:`, with the actual allowed keywords. Alternatively I'd just remove it and write a description instead:
```suggestion
    For each trainable parameter keyword arguments must be passed for the initial value 
    ``xxx`` (tensor), the numerical bounds ``xxx_bounds`` (float, float), whether the 
    parameter ``xxx`` will be optimized ``xxx_trainable`` (bool), along with any other parameters.
```
Couldn't encompass the suggestion for the last two args.